test/metrics/index.spec.js

```diff
@@ -0,0 +1,206 @@
+'use strict'
+/* eslint-env mocha */
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+chai.use(require('chai-as-promised'))
+const { expect } = chai
+
+const { randomBytes } = require('libp2p-crypto')
+const duplexPair = require('it-pair/duplex')
+const pipe = require('it-pipe')
+const concat = require('it-concat')
+const pushable = require('it-pushable')
+const { consume } = require('streaming-iterables')
+const delay = require('delay')
+
+const Metrics = require('../../src/metrics')
+const { createPeerId } = require('../utils/creators/peer')
+
+describe('Metrics', () => {
+  let peerId
+  let peerId2
+
+  before(async () => {
+    [peerId, peerId2] = await createPeerId({ number: 2, fixture: true })
```

Comment:
```suggestion
    [peerId, peerId2] = await createPeerId({ number: 2 })
```

`fixture: true` is the default